examples/supplychain-app/quorum/smartContracts/productContract.sol

```diff
@@ -0,0 +1,73 @@
+pragma solidity 0.6.1;
+
```

Comment:
Comments are needed in code.

Reply:
@Rayas-NG  Still missing comments in the code. https://www.bitdegree.org/learn/solidity-syntax
It is not needed that every line should have a comment, but the class and the functions should tell what they are about.